x-pack/plugins/osquery/public/action_results/use_action_results.ts

```diff
@@ -122,4 +125,10 @@ export const useActionResults = ({
       enabled: !skip && !!agentIds?.length,
     }
   );
+
+  if (response.error) {
+    toasts.addError(response.error as Error, { title: 'Error while fetching action results' });
```

Comment:
can we use `onError` for that? and i18n the title?

Reply:
that is a far smarter idea. thanks!